packages/table/src/TableHead/TableHead.styles.tsx

```diff
@@ -1,19 +1,53 @@
-import { css } from '@leafygreen-ui/emotion';
+import { transparentize } from 'polished';
+
+import { css, cx } from '@leafygreen-ui/emotion';
 import { Theme } from '@leafygreen-ui/lib';
 import { palette } from '@leafygreen-ui/palette';
+import { transitionDuration } from '@leafygreen-ui/tokens';
 
-export const stickyStyles = css`
-  position: sticky;
-  z-index: 1;
-  top: 0;
-`;
 export const themeStyles: Record<Theme, string> = {
   [Theme.Dark]: css`
     background-color: ${palette.black};
-    box-shadow: 0 4px ${palette.gray.dark2};
   `,
   [Theme.Light]: css`
     background-color: ${palette.white};
-    box-shadow: 0 4px ${palette.gray.light2};
   `,
 };
+
+export const getBaseStyles = (isSticky = false, theme: Theme) =>
+  cx(
+    {
+      [css`
+        position: sticky;
+        z-index: 1;
+        top: 0;
+
+        table & {
+          :after {
+            content: '';
+            position: absolute;
+            z-index: -1;
+            bottom: 0;
+            left: 0;
+            right: 0;
+            margin: auto;
+            width: 96%;
+            height: 50%;
+            box-shadow: ${theme === Theme.Light
+              ? `0px 9px 20px 9px ${transparentize(0.8, palette.black)}`
+              : '0px 6px 30px 17px rgba(0, 0, 0, 0.5)'};
+            border-radius: 40%;
+            opacity: 0;
+            transition: opacity ${transitionDuration.default}ms ease-in-out;
```

Comment:
What do you mean by inline?

Ln25 is targeting `table thead:after`. Instead of using `table` i could replace that with a className. 

Ln45 is targeting `table[data-is-sticky='true'] thead:after`

Reply:
ah nvm you're right